src/send-and-sync.md

```diff
@@ -1,13 +1,30 @@
+<!--
 # Send and Sync
+-->
 
+# Send と Sync
+
+<!--
 Not everything obeys inherited mutability, though. Some types allow you to
 multiply alias a location in memory while mutating it. Unless these types use
 synchronization to manage this access, they are absolutely not thread safe. Rust
 captures this through the `Send` and `Sync` traits.
+-->
+
+すべてのものが継承可変性に従っているわけではありません。が、いくつかの型においては、
+メモリ上の場所の値を変更している間に、複数のエイリアスを生成することが可能です。
+これらの型が、このアクセスを管理するために同期を行なわない限り、これらは絶対スレッドセーフでは
+ありません。 Rust ではこれを、 `Send` トレイトと `Sync` トレイトでキャプチャしています。
```

Comment:
`Sync` トレイトで表しています。
or
捕捉しています。
?

Reply:
"capture -> キャプチャ" is in the translation table.

Reply:
My understanding is the translation table is for the programming terms whereas this "capture" is used in a non-programming context. I thought it would be less confusing if we did not use "キャプチャ".

I'm fine either way.

Reply:
On second thought, "`Sync` トレイトで表しています。" seems better. Actually this "capture" is not in a non-programming context.
(btw is there something which should be translated into "キャプチャ"? I can't think of any words. Rust doesn't have try-catch...)

Reply:
> is there something which should be translated into "キャプチャ"? I can't think of any words. Rust doesn't have try-catch...

great question.  I actually don't know. Maybe we can say "x captures y" in
```
let opt = Some(y);
match opt {
  Some(x) => blah,
  ...
}
```
?